### ISSUE TITLE
Vim Script and Vim script should be combined

### ISSUE THREAD

**Issue by louwers** [madnight/githut#115]

**Comment by madnight** [comment]
I see, thanks for reporting, will be fixed soon.

**Comment by madnight** [comment]
I added a fix that filters the lowercase "Vim script" and leaves "Vim Script." I think "Vim Script" is the official name on GitHub, and I'm not sure why there is also a lowercase variant in the GitHub BigQuery Dataset. Currently, I do not have enough credits/dollars left to rerun the entire history with a proper merge in SQL. Therefore, I filtered out the lowercase naming, which also has a significantly lower count of events.

**Comment by louwers** [comment]
The official name is "Vim script" as per the manual.

<img width="697" alt="image" src="https://github.com/madnight/githut/assets/649392/8db32e77-4c71-4e10-bea1-99573f22b919">

**Comment by madnight** [comment]
You are right, but I referred to the "official name on GitHub". For instance if you open one of the most popular Vim script projects on GitHub and look at the language name https://github.com/tpope/vim-fugitive it will read "Vim Script".

**Comment by louwers** [comment]
Ah right, so maybe we should file a bug report with GitHub.
